test/e2e/config/docker.yaml

```diff
@@ -199,10 +199,10 @@ variables:
   # allowing the same e2e config file to be re-used in different Prow jobs e.g. each one with a K8s version permutation.
   # The following Kubernetes versions should be the latest versions with already published kindest/node images.
   # This avoids building node images in the default case which improves the test duration significantly.
-  KUBERNETES_VERSION_MANAGEMENT: "v1.23.3"
-  KUBERNETES_VERSION: "v1.23.3"
-  KUBERNETES_VERSION_UPGRADE_FROM: "v1.22.4"
-  KUBERNETES_VERSION_UPGRADE_TO: "v1.23.3"
+  KUBERNETES_VERSION_MANAGEMENT: "v1.23.4"
+  KUBERNETES_VERSION: "v1.23.4"
+  KUBERNETES_VERSION_UPGRADE_FROM: "v1.23.3"
```

Comment:
This should be 1.22.7

Reply:
@sbueringer `KUBERNETES_VERSION_UPGRADE_FROM=1.22.7` is this correct